R/actions.R

```diff
@@ -19,15 +19,15 @@ use_github_actions <- function() {
   new <- use_action_check_release()
 
   if (!new) {
-    return(invisible(FALSE))
+    return(invisible(new))
   }
-  use_github_actions_badge("R")
+  use_github_actions_badge("R-CMD-check")
 
-  invisible(TRUE)
+  invisible(new)
```

Comment:
If this is in response to my comment, it wasn't about this instance. Rather, my comment was about lines 41-45.

Reply:
It is the same point though, they are both changed now